### PR TITLE
don't disable optarch for Clang 11.0.0

### PR DESCRIPTION
**Pull request by bartoldeman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by bartoldeman** [comment]
@boegelbot please test @ generoso EB_ARGS="--include-easyblocks-from-pr 2264"

**Comment by bartoldeman** [comment]
This needs https://github.com/easybuilders/easybuild-easyblocks/pull/2264

**Comment by Micket** [comment]
@bartoldeman Any comment on the issue with optarch? Is the comment simply wrong?

**Comment by bartoldeman** [comment]
> @bartoldeman Any comment on the issue with optarch? Is the comment simply wrong?

@Micket the comment is obsolete, it's copied from an earlier version but I had no issue building it with that line removed.

**Comment by bartoldeman** [comment]
relevant commit for optarch:
fb453ae2063

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2264
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/18c997407aa03ea758afdf3d2cc96b25 for a full test report.

**Comment by Micket** [comment]
I was wrong, it was very quick. Hmm
```
/local/EB/build/Clang/11.0.0/GCCcore-9.3.0/llvm-11.0.0.src/tools/lldb/bindings/./python/python-typemaps.swig:484: Error: Syntax error in input(3).
```

```
481 struct Py_buffer_RAII {
482   Py_buffer buffer = {};
483   Py_buffer_RAII() {};
484   Py_buffer &operator=(const Py_buffer_RAII &) = delete;
485   Py_buffer_RAII(const Py_buffer_RAII &) = delete;
486   ~Py_buffer_RAII() {
487     if (buffer.obj)
488       PyBuffer_Release(&buffer);
489   }
490 };
```
Looks fine to me, so I'm not sure what goes wrong here.

**Comment by bartoldeman** [comment]
@micket perhaps it needs a build dep on swig, I my log I have:
```
-- Found SWIG: /cvmfs/soft.computecanada.ca/gentoo/2020/usr/bin/swig (found suitable version "3.0.12", minimum required is "2.0") 
```
perhaps you picked up an OS SWIG that is now too old.

**Comment by bartoldeman** [comment]
I added the SWIG builddep because it's a good idea in any case.

File: easybuild/easyconfigs/c/Clang/Clang-11.0.0-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -55,11 +56,13 @@ builddependencies = [
     ('CMake', '3.16.4'),
     ('libxml2', '2.9.10'),
     ('Python', '3.8.2'),
+    ('SWIG', '4.0.1'),
```
Comment (Micket) [inline easybuild/easyconfigs/c/Clang/Clang-11.0.0-GCCcore-9.3.0.eb:59]: perhaps also add a comment stating that it's a dep for lldb?
```suggestion
    ('SWIG', '4.0.1'),  # Optional dependency for LLDB
```

**Review by Micket — CHANGES_REQUESTED** [review]

**Review by Micket — DISMISSED** [review]
lgtm

**Comment by bartoldeman** [comment]
@boegelbot please test @ generoso EB_ARGS="--include-easyblocks-from-pr 2264"

**Comment by bartoldeman** [comment]
I asked the always friendly looking bot to do another build with the SWIG builddep

**Comment by Micket** [comment]
Test report by @Micket
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2264
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/189e66058b6e606f9ed00c1e03d44019 for a full test report.

**Comment by Micket** [comment]
@bartoldeman So.. Python turns out to be a runtime dependency;
```
$ lldb
lldb: error while loading shared libraries: libpython3.8.so.1.0: cannot open shared object file: No such file or directory
```

**Comment by Micket** [comment]
So, options as i see them
1. Just have python as a dependency here. Hurts a bit for a compiler though, since it means no multi-deps for anything using Clang.
2. Try to separate LLDB into a different module (probably painful) and let that module depend on Python.
3. Disable SWIG (we should probably enhance the easyblock to explicitly disable SWIG if no SWIG-dep is used regardless). 

I assume python dep comes from SWIG? If not, then, well, option 3 just disappears.

**Comment by bartoldeman** [comment]
Maybe 2 (seperate lldb) is not too hard. Can give that a go.

**Comment by bartoldeman** [comment]
11822 does LLDB now. Though we can still remove the optarch line I guess.

**Comment by verdurin** [comment]
@bartoldeman what's the latest on this one?

**Comment by bartoldeman** [comment]
still the same, I forgot about this one :(. Will adjust

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3501.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/c0a3e5e706d079a452c1225eb533affe for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2641.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/24739a83ed6099ce78dc782377689c97 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by boegel** [comment]
Going in, thanks @bartoldeman!

**Pull request merged**